java/checkstyle.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
+          "https://checkstyle.org/dtds/configuration_1_3.dtd">
+
+<!--
+    Checkstyle configuration that checks the Google coding conventions from Google Java Style
+    that can be found at https://google.github.io/styleguide/javaguide.html
+
+    Checkstyle is very configurable. Be sure to read the documentation at
+    http://checkstyle.sf.net (or in your downloaded distribution).
+
+    To completely disable a check, just comment it out or delete it from the file.
+
+    Authors: Max Vetrenko, Ruslan Diachenko, Roman Ivanov.
+ -->
+
+<module name = "Checker">
+    <module name="TreeWalker">
+        <module name="RegexpOnFilename">
+        <!-- For inclusive code -->
+            <property name="fileNamePattern" value="(?i)white[-_ ]*list|black[-_ ]*list|master|slave|redline|grandfathered|sanity[-_ ]*check|crippled|dummy|man[-_ ]*in[-_ ]*the[-_ ]*middle|first[-_ ]*class[-_ ]*citizen"/>
```

Comment:
where do the exceptions come in?

Reply:
Looks like it would be "suppressions" - https://checkstyle.sourceforge.io/config_filters.html#SuppressionFilter